src/screens/Learn/Learn.style.tsx

```diff
@@ -52,18 +58,131 @@ export const LearnHeading1 = styled(Heading1)`
   margin: 0 0 1.5rem;
 
   @media (min-width: ${materialSMBreakpoint}) {
-    margin: 1.5rem 0 2.5rem;
+    margin: 1.15rem 0 1.75rem;
   }
 `;
 
 export const SmallSubtext = styled(MarkdownContent)`
-  margin-bottom: 2rem;
+  margin: 0;
 
   p {
-    font-size: 14px;
+    font-style: italic;
+    margin: 0;
   }
 `;
 
 export const LastUpdatedDate = styled(Paragraph)`
   font-size: 0.875rem;
 `;
+
+/* Used for case studies landing + covid explained landing: */
+export const CardsContainer = styled(Grid).attrs(props => ({
+  container: true,
+  alignItems: 'stretch',
+}))`
+  margin-bottom: ${theme.spacing(3)}px;
+  margin-top: 0.75rem;
+  &:last-child {
+    margin-bottom: 0;
+  }
+`;
+
+/*
+ TODO (Chelsi): we're almost always removing the underline
+ manually. Lets put this styled Link somewhere global ::
+*/
```

Comment:
Actually, the best practice is to have links underlined for accessibility (they don't have enough contrast), we can probably revisit this with Victoria (but true, we might as well style this globally)